YSE_App/tests/test_chime.py

```diff
@@ -0,0 +1,36 @@
+""" Tests for CHIME """
+import os, sys
+import numpy as np
+
+import pandas 
+
+try:
+    sys.path.append(os.path.abspath("../galaxies"))
+    import path
+except ModuleNotFoundError:
+    from YSE_App.galaxies import path
+
+from IPython import embed
+
+def internal_test_of_path():
+    """ Check PATH analysis internally (i.e. not with the DB)"""
+    # Load up the table
+    chime_frbs = pandas.read_csv('../chime/chime_tests.csv')
```

Comment:
Error `FileNotFoundError: [Errno 2] No such file or directory: '../chime/chime_tests.csv'` when running from shell

Reply:
this is deprecated;  am removing the file